src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java

```diff
@@ -0,0 +1,395 @@
+package pl.edu.pw.mini.msi.knowledgerepresentation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.ActionDomain;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentenceParts.*;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentences.*;

```

Comment:
\[Checkstyle\] WARNING: Using the '\.\*' form of import should be avoided \- pl\.edu\.pw\.mini\.msi\.knowledgerepresentation\.actionDomain\.sentences\.\*\.